runtime/doc/lua.txt

```diff
@@ -707,6 +707,38 @@ vim.mpack.encode({obj})                                 *vim.mpack.encode*
 vim.mpack.decode({str})                                 *vim.mpack.decode*
         Decodes (or "unpacks") the msgpack-encoded {str} to a Lua object.
 
+------------------------------------------------------------------------------
+VIM.SPELL                                                           *lua-spell*
+
+vim.spell.check({str}, {opts})                           *vim.spell.check*
```

Comment:
It looks like `{opts}` is unused. Since this is not part of the API there is no strict contract preventing us from adding an argument in the future if need be, so we don't need to "defensively" add an `opts` table here like we often do for API functions.

Reply:
I was supposed to remove this. I was going to add all equivalents to all the `spell*` related options but decided it wasn't worth the effort.